src/pyfmi/fmi3.pyx

```diff
@@ -1912,6 +1913,33 @@ cdef class FMUModelBase3(FMI_BASE.ModelBase):
 
         return variable_dict
 
+    def get_derivatives_list(self):
+        """
+        Returns a dictionary with the states derivatives.
+
+        Returns::
+
+            An ordered dictionary with the derivative variables.
+        """
+        cdef FMIL3.fmi3_import_variable_list_t* variable_list
+        cdef FMIL.size_t                        variable_list_size
+        cdef FMIL3.fmi3_import_variable_t*      variable
+        variable_dict = {}
+
+        variable_list = FMIL3.fmi3_import_get_continuous_state_derivatives_list(self._fmu)
+        if variable_list == NULL:
+            raise FMUException("The returned state_derivatives list is NULL.")
```

Comment:
I think we should make this error message more clear while we are here, considering this is what a user potentially sees when invoking `fmu.get_derivatives_list()`.

Reply:
Any suggestions? 

Practically speaking, this is static information from the `modelDescription.xml`, so I have a hard time seeing how this will fail, other than a bug in FMILibrary.

Reply:
Ahh I see, I thought it was just that there are no state derivatives, but perhaps it is just an empty list then? Could we maybe simply write something in the line of 'Internal error when retrieving the state derivatives, please contact support.' or similar? I just thought that it could at least be good if we give a hint of what to do if a user would see that error.